<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemProfWidget</class>
 <widget class="QWidget" name="MemProfWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1221</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Memory profiler</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="vertLayout">
     <item>
      <widget class="QLabel" name="labelStatus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>disconnected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCustomPlot" name="plot" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="snapshotProgress">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horzLayout">
       <item>
        <widget class="QTableView" name="generalStatTable">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="allocPoolTable">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="tagTable">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="snapshotList">
         <property name="selectionMode">
          <enum>QAbstractItemView::MultiSelection</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="QtIcons.qrc"/>
 </resources>
 <connections/>
</ui>
